docs-kits/kits/Supply Chain Disruption Notifications KIT/page_software-development-view.md

```diff
@@ -0,0 +1,275 @@
+submodel API.
+
+### Usage Policies[​](https://eclipse-tractusx.github.io/docs-kits/kits/Traceability%20Kit/Software%20Development%20View/Data%20Provider%20Development%20View%20Traceability%20Kit/#usage-policies "Direct link to heading")
+
+To decide which company can use the data asset under specific conditions, usage policies (also referred to as contract
+policies) are used. Therefore, they are more specific than access policies and only used just after access is granted.
+Currently, the usage policies aren't technically enforced but based on a legal framework agreements. Signing of
+framework agreements can be enforced during negotiation depending on the connector implementation.
+
+Policies are defined based on the [W3C ODRL format](https://www.w3.org/TR/odrl-model/). This allows a standardized way
+of formulating policy payloads. It further allows to stack different constraints with the `odrl:and` operator.
+Therefore, every data provider can decide on his or her own under which conditions their data assets are shared in the
+network. It is recommended to restrict the data usage for all exchanged data standards. An example of one usage policy
+containing three different constraints is shown and described in the following:
+
+```json
+{
+  "@context": [
+    "http://www.w3.org/ns/odrl.jsonld",
+    {
+      "edc": "https://w3id.org/edc/v0.0.1/ns/",
+      "cx-policy": "https://w3id.org/catenax/policy/"
+    }
+  ],
+  "@type": "PolicyDefinitionRequestDto",
+  "@id": "<POLICY-ID>",
+  // Important for the contract definition
+  "edc:policy": {
+    "@type": "Set",
+    "profile": "cx-policy:profile2405",
+    "permission": [
+      {
+        "action": "use",
+        "constraint": {
+          "@type": "LogicalConstraint",
+          "and": [
+            {
+              "@type": "LogicalConstraint",
+              "leftOperand": "cx-policy:FrameworkAgreement",
+              "operator": "eq",
+              "rightOperand": "<FRAMEWORK-AGREEMENT>"
+            },
+            {
+              "@type": "LogicalConstraint",
+              "leftOperand": "cx-policy:UsagePurpose",
+              "operator": "eq",
+              "rightOperand": "<USAGE-PURPOSE>"
+            },
+            {
+              "@type": "LogicalConstraint",
+              "leftOperand": "cx-policy:ContractReference",
+              "operator": "eq",
+              "rightOperand": "x12345"
+            }
+          ]
+        }
+      }
+    ]
+  }
+}
+```
+
+It is recommended to use the following values for rightOperand of FrameworkAgreement and UsagePurpose depending on the
+use case:
+
+| Use case | cx-policy:FrameworkAgreement | cx-policy:UsagePurpose |
+| -------- | ---------------------------- | ---------------------- |
+| DCM      | `DemandCapacity:1.0`         | `cx.dcm.base:1`        |
+| PURIS    | `Puris:1.0`                  | `cx.puris.base:1`      |
```

Comment:
Did someone checked this part? In my understanding the FrameworkAgreement "Puris:1.0" as well as "DemandCapacity:1.0" will be deprecated with the release of Jupiter.

see: https://github.com/catenax-eV/cx-odrl-profile/blob/main/profile.md

> Puris:1.0
> Valid from: 2024-06-20
> Valid until: 2024-10-16
> Status: deprecated

Reply:
@tom-rm-meyer-ISST can you check this part?

Reply:
Same as in PURIS KIT discussion. We'll keep this as updates on use case related content has been planned for R24.12.